README.md

```diff
@@ -5,6 +5,10 @@ basic chome extension to automatically update cryptocurrency holdings (any coin:
 
 ## Changelog
 
+#### 12/14/17:
+
+* Address support for ERC20 Tokens. Put the wallet address in the "Account Description" field in the format "erc20:address".
```

Comment:
Add a note here so people know that their address must start with "0x" for ethplorer. The blockcypher API doesn't care if its added or not.

Reply:
I added support for when the 0x is missing